### PR TITLE
Add operation to the object

### PR DESCRIPTION
**Pull request by maen-al-khraisha**
new features:

1. add object
2. delete objects.
3. update object
4. and other utility functions

### CODE REVIEW

File: src/module/meme-object.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+const add = (item) => {
+  memeList = getLocal();
+  const len = memeList.length;
+  const obj = { description: item, completed: false, index: len + 1 };
+  memeList.push(obj);
+  setLocal();
+};
+
+// === get meme list === //
+const getLen = () => getLocal().length;
```
Comment (AndreLau11) [inline src/module/meme-object.js:48]: the getLen variable seem to be to confused so please consider to change it to a more specific name.

File: src/module/meme-object.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  id -= 1;
+  memeList[id].description = text;
+  setLocal();
+};
+
+// === delete item from meme === //
+const del = (index) => {
+  index = Number(index);
+  memeList = getLocal();
+  memeList = memeList.filter((e) => e.index !== index);
```
Comment (AndreLau11) [inline src/module/meme-object.js:34]: Please consider changing the "e" for the complete word event.

**Review by AndreLau11 — CHANGES_REQUESTED** [review]
Good job so far please consider making the following messages.

File: src/module/meme-object.js (hunk 1 above)
Comment (maen-al-khraisha) [reply src/module/meme-object.js:48]: yes sure 💯

File: src/module/meme-object.js (hunk 2 above)
Comment (maen-al-khraisha) [reply src/module/meme-object.js:34]: thank you 😃

**Pull request merged**